project/src/test/java/com/google/sps/PrefilledInformationTest.java

```diff
@@ -0,0 +1,55 @@
+package com.google.sps;
+
+import com.google.sps.servlets.PrefilledInformationServlet;
+import com.google.sps.utilities.CommonUtils;
+import com.google.sps.utilities.PrefilledInformationConstants;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringWriter;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+import org.mockito.Mockito;
+
+/** */
+@RunWith(JUnit4.class)
+public final class PrefilledInformationTest {
+  private static final String REQUEST_CATEGORY = "category";
+  private static final String INTERESTS = "interests";
+  private static final String SKILLS = "skills";
+
+  @Test
+  public void requestInterests() throws IOException {
+    // Verify that returned interests is the provided interest constant
+    HttpServletRequest request = Mockito.mock(HttpServletRequest.class);
```

Comment:
Move the common part to a SetUp funciton.
https://docs.google.com/document/d/1M9kKw-Qgwmi-fhIDab-HCsK6sp53qRJQaM7aM3s4rXs/edit#heading=h.9f87use2w5ox

Reply:
Done